android/app/src/main/java/app/covidshield/receiver/worker/ExposureCheckNotificationWorker.kt

```diff
@@ -0,0 +1,96 @@
+package app.covidshield.receiver.worker
+
+import android.annotation.TargetApi
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import android.text.Html
+import android.text.Spanned
+import android.util.Log
+import androidx.core.app.NotificationCompat
+import androidx.work.CoroutineWorker
+import androidx.work.ForegroundInfo
+import androidx.work.WorkerParameters
+import app.covidshield.MainActivity
+import app.covidshield.R
+import com.facebook.react.ReactApplication
+import com.facebook.react.modules.core.RCTNativeAppEventEmitter
+import kotlinx.coroutines.delay
+
+
+class ExposureCheckNotificationWorker (private val context: Context, parameters: WorkerParameters) :
+        CoroutineWorker(context, parameters) {
```

Comment:
Formatting here: remove the indent?

Reply:
Not according to "Reformat Code" option in Android Studio. I think it's because it's an extension of the line above.